nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -254,6 +254,14 @@ in {
       '';
     };
 
+    fastcgiTimeout = mkOption {
+      type = types.int;
+      default = 120;
+      description = ''
```

Comment:
```suggestion
      description = mdDoc ''
```
writing new module documentation as docbook is discouraged.

Reply:
but, there's no docbook in it?

Reply:
it'll be parsed as docbook regardless, and the more options we can *not* parse as docbook when they're introduced the easier the migration away from docbook will be